portal-api/src/v1/controllers/deal-status/send-status-update-emails.js

```diff
@@ -1,9 +1,11 @@
 const sendEmail = require('../../email');
 
 const sendEmailsToOwningBanks = async (templateId, emailVariables, owningBankEmails) => {
-  owningBankEmails.forEach(async (email) => {
-    await sendEmail(templateId, email, emailVariables);
-  });
+  await Promise.all(
+    owningBankEmails.map(async (email) => {
+      return await sendEmail(templateId, email, emailVariables);
+    }),
```

Comment:
The `{}` can be removed here to just be `owingBankEmails.map(async (email) => await sendEmail(templateId, email, emailVariables))`

Reply:
great spot